api/test/resources/workflows/sarscov2_illumina_full/sarscov2_illumina_full.wdl

```diff
@@ -0,0 +1 @@
+https://cdn.jsdelivr.net/gh/broadinstitute/viral-pipelines@master/pipes/WDL/workflows/sarscov2_illumina_full.wdl
```

Comment:
same question as the above: should we pin to a commit/tag of the broadinstitute/viral-pipelines repo?